docs/source/mistral.rst

```diff
@@ -126,4 +126,48 @@ There are more workflow examples under :github_st2:`/usr/share/doc/st2/examples
 
 Check out this step-by-step tutorial on building a workflow in |st2| http://stackstorm.com/2015/07/08/automating-with-mistral-workflow/
 
-And more details on Mistral workflow syntax on https://wiki.openstack.org/wiki/Mistral/DSLv2
\ No newline at end of file
+And more details on Mistral workflow syntax on https://wiki.openstack.org/wiki/Mistral/DSLv2
+
+Configuration Options
```

Comment:
This should probably go to the installation / configuration section.

Currently, this document contains end user docs so putting configuration info here might make it confusing...